status/format.go

```diff
@@ -168,18 +168,26 @@ func FormatDetails(taskRun *helpers.TaskRun) (string, error) {
 		return "", nil
 	}
 
+	if result.ValidationError != nil {
+		return fmt.Sprintf("Invalid TEST_OUTPUT result: %s", result.ValidationError), nil
```

Comment:
in Github, it will be reported as failed test and in details will be message what's wrong with TEST_OUTPUT